packages/theming/atlas/src/themesource/atlas_core/native/core/widgets/radiobuttons.ts

```diff
@@ -18,36 +18,46 @@ const RADIO_BUTTON_INNER_CIRCLE_SIZE = 8;
 
 export const com_mendix_widget_native_radiobuttons_RadioButtons: RadioButtonsStyles = {
     labelTextStyle: {
+        // All TextStyle properties are allowed
         color: font.colorTitle,
         fontSize: font.sizeSmall,
         lineHeight: font.lineHeightSmall,
         marginBottom: spacing.small
     },
     radioButtonItemContainerStyle: {
+        // All ViewStyle properties are allowed
         marginBottom: spacing.small
     },
     radioButtonItemContainerHorizontalStyle: {
+        // All ViewStyle properties are allowed
         marginEnd: spacing.small
     },
     circularButtonStyle: {
+        // width, height, borderRadius, borderColor and All ViewStyle properties are allowed
         width: RADIO_BUTTON_OUTER_CIRCLE_SIZE,
         height: RADIO_BUTTON_OUTER_CIRCLE_SIZE,
         borderRadius: RADIO_BUTTON_OUTER_CIRCLE_SIZE / 2,
         borderColor: border.color,
```

Comment:
I see you introduced variables and custom-variables, however, you haven't used the variable values here, in this object. If you reference them here, then end-users can change the `custom-variables` file and the changes will reflect here.

Reply:
done

Reply:
Howcome some properties use the widget's variables and other properties use e.g. "border" and "spacing" variables?

Reply:
The goal here is to use all the widget's variables

Reply:
Otherwise end-users can't _easily_ customise things

Reply:
can you check now?